cmd/promtool/main.go

```diff
@@ -134,6 +134,12 @@ func main() {
 	dumpMinTime := tsdbDumpCmd.Flag("min-time", "Minimum timestamp to dump.").Default(strconv.FormatInt(math.MinInt64, 10)).Int64()
 	dumpMaxTime := tsdbDumpCmd.Flag("max-time", "Maximum timestamp to dump.").Default(strconv.FormatInt(math.MaxInt64, 10)).Int64()
 
+	importCmd := tsdbCmd.Command("create-blocks-from", "[Experimental] Import samples from input and produce TSDB blocks. Please refer to the storage docs for more details.")
+	openMetricsImportCmd := importCmd.Command("openmetrics", "Import samples from OpenMetrics input and produce TSDB blocks. Please refer to the storage docs for more details.")
+	// TODO: add flag to set default block duration
+	importFilePath := openMetricsImportCmd.Arg("input file", "Imports samples from file instead of stdin.").String()
```

Comment:
"instead of stdin" is confusing, as we don't support stdin (for now). I'd make this Arg `Required()` and just use "OpenMetrics file to read samples from" as a doc string.